aicssegmentation/workflow/structure_wrapper_config.py

```diff
@@ -147,5 +147,7 @@ def _workflow_decoder(self, obj: Dict, workflow_name: str) -> WorkflowDefinition
                 step.parameter_defaults = param_defaults
 
             steps.append(step)
-
-        return WorkflowDefinition(name=workflow_name, steps=steps)
+        if from_file:
```

Comment:
I'd like to see a unit test for this - testing that we get the right kind of WorkflowDefinition object depending on whether `from_file` is true or not. This method is now public so makes sense to test it.